### DISCUSSION TITLE
Unusual create-next-app warnings

### DISCUSSION THREAD

**Discussion by feechin** [vercel/next.js#33743] [Help]
Hello,

I am getting a lot of warnings on using create-next-app.  I tried uninstalling node and npm completely and and reinstalled but nothing works.  I wonder if anyone can help me understand why this happened:
node v16.8.0
macOS Monterey

Note: I have another mac with node v16.8.0 and have no warnings with create-next-app.  The results were clean.

`npx create-next-app@latest hello7

Using npm.

Installing dependencies:
- react
- react-dom
- next

npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'postcss@8.4.5',
npm WARN EBADENGINE   required: { node: '^10 || ^12 || >=14' },
npm WARN EBADENGINE   current: { node: 'v13.6.0', npm: '7.21.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'nanoid@3.2.0',
npm WARN EBADENGINE   required: { node: '^10 || ^12 || ^13.7 || ^14 || >=15.0.1' },
npm WARN EBADENGINE   current: { node: 'v13.6.0', npm: '7.21.0' }
npm WARN EBADENGINE }

added 16 packages, and audited 17 packages in 5s

2 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

Installing devDependencies:
- eslint
- eslint-config-next

npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'nanoid@3.2.0',
npm WARN EBADENGINE   required: { node: '^10 || ^12 || ^13.7 || ^14 || >=15.0.1' },
npm WARN EBADENGINE   current: { node: 'v13.6.0', npm: '7.21.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'postcss@8.4.5',
npm WARN EBADENGINE   required: { node: '^10 || ^12 || >=14' },
npm WARN EBADENGINE   current: { node: 'v13.6.0', npm: '7.21.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'eslint@8.7.0',
npm WARN EBADENGINE   required: { node: '^12.22.0 || ^14.17.0 || >=16.0.0' },
npm WARN EBADENGINE   current: { node: 'v13.6.0', npm: '7.21.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@eslint/eslintrc@1.0.5',
npm WARN EBADENGINE   required: { node: '^12.22.0 || ^14.17.0 || >=16.0.0' },
npm WARN EBADENGINE   current: { node: 'v13.6.0', npm: '7.21.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'eslint-scope@7.1.0',
npm WARN EBADENGINE   required: { node: '^12.22.0 || ^14.17.0 || >=16.0.0' },
npm WARN EBADENGINE   current: { node: 'v13.6.0', npm: '7.21.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'eslint-utils@3.0.0',
npm WARN EBADENGINE   required: { node: '^10.0.0 || ^12.0.0 || >= 14.0.0' },
npm WARN EBADENGINE   current: { node: 'v13.6.0', npm: '7.21.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'eslint-visitor-keys@3.2.0',
npm WARN EBADENGINE   required: { node: '^12.22.0 || ^14.17.0 || >=16.0.0' },
npm WARN EBADENGINE   current: { node: 'v13.6.0', npm: '7.21.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'espree@9.3.0',
npm WARN EBADENGINE   required: { node: '^12.22.0 || ^14.17.0 || >=16.0.0' },
npm WARN EBADENGINE   current: { node: 'v13.6.0', npm: '7.21.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@typescript-eslint/parser@5.10.1',
npm WARN EBADENGINE   required: { node: '^12.22.0 || ^14.17.0 || >=16.0.0' },
npm WARN EBADENGINE   current: { node: 'v13.6.0', npm: '7.21.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@typescript-eslint/scope-manager@5.10.1',
npm WARN EBADENGINE   required: { node: '^12.22.0 || ^14.17.0 || >=16.0.0' },
npm WARN EBADENGINE   current: { node: 'v13.6.0', npm: '7.21.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@typescript-eslint/types@5.10.1',
npm WARN EBADENGINE   required: { node: '^12.22.0 || ^14.17.0 || >=16.0.0' },
npm WARN EBADENGINE   current: { node: 'v13.6.0', npm: '7.21.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@typescript-eslint/typescript-estree@5.10.1',
npm WARN EBADENGINE   required: { node: '^12.22.0 || ^14.17.0 || >=16.0.0' },
npm WARN EBADENGINE   current: { node: 'v13.6.0', npm: '7.21.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@typescript-eslint/visitor-keys@5.10.1',
npm WARN EBADENGINE   required: { node: '^12.22.0 || ^14.17.0 || >=16.0.0' },
npm WARN EBADENGINE   current: { node: 'v13.6.0', npm: '7.21.0' }
npm WARN EBADENGINE }

added 203 packages, and audited 220 packages in 11s

63 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

Initialized a git repository.

Success! Created hello7 at ....
`

**Comment by ajTronic** [comment]
Same here but for create react app. I think it has to do with the node.js version. 
See https://stackoverflow.com/questions/70269056/cause-of-npm-warn-ebadengine
